src/tools/rbd/action/Nbd.cc

```diff
@@ -21,6 +21,9 @@ namespace po = boost::program_options;
 static int call_nbd_cmd(const po::variables_map &vm,
                         const std::vector<std::string> &args,
                         const std::vector<std::string> &ceph_global_init_args) {
+  #ifdef _WIN32
```

Comment:
This file already has hooks for FreeBSD builds -- should Windows just mimic that?